packages/visual-editor/src/components/Dropzone/useComponentProps.ts

```diff
@@ -131,6 +131,8 @@ export const useComponentProps = ({
             ...acc,
             [variableName]: value,
           };
+        } else {
+          return { ...acc };
```

Comment:
Is this only temporary until we implement proper behavior for handling the hyperlink value?

Reply:
yes exactly, this will be implemented in the next PR